samples/nrf9160/modem_shell/README.rst

```diff
@@ -27,6 +27,9 @@ MoSh uses the LTE link control driver to establish an LTE connection and initial
 
 The subsections list the MoSh features and show shell command examples for their usage.
 
+.. note::
+   Some of the information is available in MoSh command usages and that information is not duplicated into this document.
```

Comment:
I would like to clarify this a bit. The previous sentence talks about "command examples for their usage", which refers to the examples in this document, while "usage" in this sentence talks about the help which can be seen by typing the command in the shell without any parameters. Here's a suggestion:
```suggestion
   Some of the information is available in MoSh command usage, that can be seen by typing the command without any parameters.
   All information is not duplicated into this document.
```